### ISSUE TITLE
[Suggestion] White/Blacklist for tweakHandRestock

### ISSUE THREAD

**Issue by platinorum** [maruohon/tweakeroo#171]
I'm playing on a server, where restocking some items isn't allowed, but I want some other blocks to be restocked. So a White/Blacklist for **tweakHandRestock** would be nice!

thx for ur great mod btw <3

**Comment by maruohon** [comment]
A white-/blacklist for hand restock is coming in the next bigger updates.

But the normal hand restock also won't even work for totems (as per the original issue text) , as the totems are not related to left or right clicking actions, which is the only case where the current "normal" hand restock happens from: either before or after a attack or use (left or right click) action.

**Comment by platinorum** [comment]
Yes, I know I thought auto-totem was a feature of tweakeroo, but it was from an other mod (~ItemScroller~ **Mouse Wheelie**). I mixed it up.. :)

**Comment by maruohon** [comment]
Item Scrolller doesn't have an auto-totem feature, nor does it have any kind of hand restocking.

**Comment by platinorum** [comment]
That's absolutely right :D i meant Mouse Wheelie
